entrypoint.sh

```diff
@@ -8,6 +8,13 @@ if [ -n "$INPUT_PRE_BUILD_COMMANDS" ]; then
   eval "$INPUT_PRE_BUILD_COMMANDS"
 fi 
 
+echo "INPUT_BUNDLER_VERSON:${INPUT_BUNDLER_VERSON}"
```

Comment:
For some reason this is coming back as empty when testing with:

```
      - uses: inverse/jekyll-action@override-bundler
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          bundler_version: '2.2.31'
          ```

Reply:
Replace `VERSON` with `VERSION` maybe?

Reply:
How did I miss that 🙈